pages/index.tsx

```diff
@@ -1,5 +1,135 @@
+import useSWR from "swr";
+import { Box, Paper, useMediaQuery } from "@mui/material";
+import { makeStyles } from "@mui/styles";
+import Image from "next/image";
+
+import { Total } from "../components/stats/Total";
+import { Classes } from "../components/stats/Classes";
+import { Destinations } from "../components/stats/Destinations";
+import { Authors } from "../components/stats/Authors";
+
+type StatsResponse = Record<string, string>;
+
+async function fetcher<JSON = any>(input: RequestInfo, init?: RequestInit): Promise<JSON> {
+  const res = await fetch(input, init);
+  return res.json();
+}
+
+const useStyles = makeStyles({
+  grid: {
+    padding: "1rem",
+    minHeight: "100vh",
+    display: "grid",
+    gridTemplateColumns: "repeat(12, 1fr)",
+    gridTemplateRows: "1fr 1fr 2fr",
+    gridGap: "1rem",
+    backgroundColor: "#eeeeee",
+  },
+});
+
 const Home = () => {
-  return <div>helo</div>;
+  const { data, error } = useSWR<StatsResponse>("/api/stats", fetcher, { refreshInterval: 5000 });
```

Comment:
2. We moved to react-query on all the other queries, so let's refactor this one too

Reply:
yep, will do. pretty sure i wrote this before we made the switch so...